plancton/__init__.py

```diff
@@ -469,9 +469,12 @@ def main_loop(self):
       self._influxdb_setup()
     running = self._count_containers()
     self.logctl.debug("CPU used: %.2f%%, available: %.2f%%" % (self.efficiency, self.idle))
-    self.streamer(series="measurement",
+    self.streamer(series="host",
                   tags={ "hostname": self._hostname },
-                  fields={ "cpu_eff": self.efficiency, "containers": running })
+                  fields={ "cpu_eff": self.efficiency })
+    self.streamer(series="daemon",
+                  tags={ "hostname": self._hostname },
+                  fields={ "containers": running })
```

Comment:
In such a way it would not be possible to show the number of running containers alongside the other daemon-related data